gtsfm/data_association/data_assoc.py

```diff
@@ -91,13 +92,12 @@ def run(
         logger.debug("[Data association] input number of tracks: %s", len(tracks_2d))
         logger.debug("[Data association] input avg. track length: %s", np.mean(track_lengths_2d))
 
-        # initializer of 3D landmark for each track
-        point3d_initializer = Point3dInitializer(
-            cameras,
-            self.mode,
-            self.reproj_error_thresh,
-            self.num_ransac_hypotheses,
-        )
+        # Initialize 3D landmark for each track
+        point3d_initializer = Point3dInitializer(cameras, self.triangulation_options)
+        #     self.mode,
+        #     self.reproj_error_thresh,
```

Comment:
commented-out code -- can it be removed?

Reply:
removed